src/main/java/org/jetbrains/bsp/bazel/server/BUILD

```diff
@@ -6,9 +6,11 @@ java_library(
     visibility = ["//visibility:public"],
     deps = [
         "//src/main/java/org/jetbrains/bsp/bazel/common",
+        "//src/main/java/org/jetbrains/bsp/bazel/server/bazel",
+        "//src/main/java/org/jetbrains/bsp/bazel/server/bazel/data:bazel-data",
+        "//src/main/java/org/jetbrains/bsp/bazel/server/bazel/utils:bazel-utils",
```

Comment:
tbh I used to use a plural form for packages, I think that we have to agree on one convention + @agluszak what is the preferred bazel package naming?

Reply:
I like ``services`` much less than ``service`` but I guess both are fine. We can decide which one we want and adjust the other packages (because now we have one ``util`` and one ``utils``).

Reply:
I agree with 1, but prefer plurals for packages

Reply:
in general we want to use implicitly generated targets for packages

Reply:
done (used plural)